### PR TITLE
Scenario vizualization update

### LINKED ISSUE #230: Feedback on scenario core viz elmt
- [ ] Title (the label) should be bigger (from @vhfgosselin)
- [x] Should be expanded by default
- [x] Sub title `custom properties` is unnecessary
- [x] Tags should be True by default
- [ ] tag edition not working?
- [x] cycle should be displayed with `scenario.cycle.get_simple_label()`
- [x] expanded does not work
- [x] expandable does not work

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
Resolves #230.

### CODE REVIEW

**Review by FabienLelaquais — DISMISSED** [review]
👌(although this should have been the first PR hu?)
Well done!

File: src/taipy/gui_core/viselements.json [hunk 1]
```diff
@@ -88,13 +88,13 @@
...
                         "default_value": "True",
-                        "doc": "If True, the scenario visualizer can be expanded.<br/>If False, the scenario visualizer is always collapsed and only its name and the <i>submit</i> button are visible."
+                        "doc": "If True, the scenario visualizer can be expanded.<br/>If False, the scenario visualizer is not expandable and it is shown depending on expanded value."
                     },
                     {
                         "name": "expanded",
                         "type": "bool",
                         "default_value": "True",
-                        "doc": "If True, when a valid scenario is selected, the scenario visualizer is expanded, and its content is displayed.<br/>If False, the scenario visualizer is collapsed and only its name a <i>submit</i> button are visible."
+                        "doc": "If True, when a valid scenario is selected, the scenario visualizer is expanded, and its content is displayed.<br/>If False, the scenario visualizer is collapsed and only its name and <i>submit</i> button are visible."
```
Comment (FabienLelaquais) [inline src/taipy/gui_core/viselements.json]: I guess you want to remove the comma in "expanded, and"

**Review by FabienLelaquais — DISMISSED** [review]
👍

**Comment by FredLL-Avaiga** [comment]
👍

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**